app/src/main/res/values/strings.xml

```diff
@@ -640,4 +641,7 @@
     <string name="feed_update_threshold_option_always_update">Always update</string>
     <string name="feed_use_dedicated_fetch_method_title">Fetch from dedicated feed when available</string>
     <string name="feed_use_dedicated_fetch_method_summary">Available in some services, it is usually much faster but may return a limited amount of items and often incomplete information (e.g. no duration, item type, no live status).</string>
+    <string name="feed_use_dedicated_fetch_method_enable_button">Enable fast mode</string>
+    <string name="feed_use_dedicated_fetch_method_disable_button">Disable fast mode</string>
+    <string name="feed_use_dedicated_fetch_method_help_text">Do you think that the feed loading is too slow? If so, try enabling fast loading (you can change it in the settings or pressing the button below).\n\nNewPipe offers two ways of loading the feed:\n• Fetching the whole subscription channel, the slow but complete.\n• Using a dedicated service endpoint, the fast but usually not complete.\n\nThe difference between the two is that the fast one usually lacks some information, like the item\'s duration or type (can\'t distinguish between live videos and normal ones) and it may return less items.\n\nYouTube is an example of a service that offers this fast method with its RSS feed.\n\nSo the choice boils down to what you prefer: speed or precise information.</string>
```

Comment:
```suggestion
    <string name="feed_use_dedicated_fetch_method_help_text">Do you think feed loading is too slow? If so, try enabling fast loading (you can change it in settings or by pressing the button below).\n\nNewPipe offers two feed loading strategies:\n• Fetching the whole subscription channel, which is slow but complete.\n• Using a dedicated service endpoint, which is fast but usually not complete.\n\nThe difference between the two is that the fast one usually lacks some information, like the item\'s duration or type (can\'t distinguish between live videos and normal ones) and it may return less items.\n\nYouTube is an example of a service that offers this fast method with its RSS feed.\n\nSo the choice boils down to what you prefer: speed or precise information.</string>
```

Reply:
Wow, forgot some words there while tweaking it. Being sleepy doesn't help as well.

Thanks.